index.html

```diff
@@ -309,5 +309,28 @@ <h3>Skills</h3>
             </div>
       </section>
     </main>
+
+    <div class="contact-container">
+      <div class="contact-container-1">
+        <h5>
+          Contact me
+        </h5>
+        <p class="tonic-par1">
+          If you have an application
+          you are interested in developing,
+          a feature that you need built or
+          a project that needs coding. 
+          I’d love to help with it 
+        </p>
+      </div>
+      <div class="container">
+        <form method="POST" action="https://formspree.io/f/mrgrarlj">
+          <input type="text" id="name" name="name" placeholder="name" maxlength="30" required/>
+          <input type="email" name="email" placeholder="email@example.com" required/>
+          <textarea id="subject" name="message" cols="30" rows="10" placeholder="message" maxlength="500" required></textarea>
+          <input type="submit" value="Get In Touch"/>
```

Comment:
- [ ] When the submit button is pressed, this should change the green color, but it looks the color change for the button is red in place green. I kindly invite you to change it according to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).
![image](https://user-images.githubusercontent.com/20563252/126505029-3f428cd2-cb9e-4246-8f76-41dd83ab7576.png)